app/controllers/episodes_controller.rb

```diff
@@ -19,28 +19,34 @@ def index
 
   # GET /episodes/1
   def show
-    authorize @episode
+    redirect_to edit_episode_url(@episode)
   end
 
   # GET /episodes/new
   def new
     @episode = Episode.new(episode_params)
+    @episode.strict_validations = true
+    @episode.valid? if turbo_frame_request?
```

Comment:
Wanna make sure @radical-ube and I are understanding this right:
So when the turbo frame receives a change to update the DOM, we want to check if the episode/audio is valid or not, and if it isn't, display the error.

Reply:
So the only reason I need to call that ... is to get the validation errors to show up on this field:

![image](https://user-images.githubusercontent.com/1410587/231875644-bb154eed-12fb-4493-84ab-2b4901c1e245.png)

When you change that field to blank, Turbo makes a request for:

```
GET http://localhost:3002/episodes/<guid>/edit?episode%5Bsegment_count%5D=
```

That's the `episodes#show` route.  (Or `episodes#new` - if you're creating a new episode).

You usually don't call `.valid?` anywhere in the `def show` or `def new` methods.  Since you don't need/want a bunch of validation messages to show up in the form before the user has even entered any data.  But here, for the turbo-request only ... I did want that validation error to show up.